webapp-client/src/component/DragDropContainer.js

```diff
@@ -1,48 +1,73 @@
 import React from 'react';
-import { useDrag } from 'react-dnd';
+import { useDrag, useDrop } from 'react-dnd';
 import PropTypes from 'prop-types';
 import allowErrorMessage from '../errorify';
 
 /**
  * Draggable category container displayed in category list
  * @param {*} props
  */
-function Drag(props) {
+function DragAndDrop(props) {
   const title = props.title;
-  const id = props.id;
+  const categoryId = props.categoryId;
+
+  // Drag action dragging categories
   const [{ isDragging }, drag] = useDrag({
     item: { title, type: 'category' },
     end: (item, monitor) => {
       const isDropped = monitor.getDropResult();
       if (item && isDropped) {
-        props.deleteCategory(id);
+        props.deleteCategory(categoryId);
       }
     },
     collect: (monitor) => ({
       isDragging: monitor.isDragging(),
     }),
   });
 
+  //Drop action accepting items
+  const [{ canDrop, isOver }, drop] = useDrop({
+    accept: 'item',
+    drop: () => ({ categoryId: categoryId }),
+    collect: (monitor) => ({
```

Comment:
nit: could monitor be replaced with cursor here?

Reply:
monitor is a key word so unfortunately no:(((

Reply:
What do you mean by monitor is a key word?